bin/idea.sh

```diff
@@ -69,6 +69,9 @@ do
   shift
 done
 
+if [ -e $IDEA_OUTPUT ] ; then
+    rm -r $IDEA_OUTPUT
+fi
```

Comment:
Was getting some weird errors when the `.idea` folder already existed when running the script, so I added this. It seems useful, so I've left it in.

Reply:
Yep - noticed that, it's good